lib/internal/options.js

```diff
@@ -38,7 +39,22 @@ function getEmbedderOptions() {
 }
 
 function generateConfigJsonSchema() {
-  const map = getEnvOptionsInputType();
+  const envOptionsMap = getEnvOptionsInputType();
+  const namespaceOptionsMap = getNamespaceOptionsInputType();
+
+  function createPropertyForType(key, type) {
```

Comment:
in this function, where is `key` get used?

Reply:
good catch!!! remove the unused prop